interface/modules/custom_modules/oe-module-faxsms/src/Controller/AppDispatch.php

```diff
@@ -545,7 +554,7 @@ public static function getLoggedInUser(): array
     public function getPatientDetails(): bool|string
     {
         $id = $this->getRequest('pid');
-        $query = "SELECT fname, lname, phone_cell, email FROM Patient_data WHERE pid = ?";
+        $query = "SELECT fname, lname, phone_cell, email FROM patient_data WHERE pid = ?";
```

Comment:
Turns out that on Linux, table names are case sensitive...

Reply:
yep, OpenEMR learned this lesson the hard way a long back on the form_CAMOS_* tables (when install the CAMOS form). If install on windows (not case sensitive for table names), then it is made to be lower case, and if then migrate that install to linux, it would then break (since linux is case sensitive). We had to create a special function mechanism so it wouldn't break the CAMOS forms: 
https://github.com/openemr/openemr/blob/3298e343649730c5fd353ff93592db980fbd3dd6/library/formdata.inc.php#L182-L192